### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by daxiondi**
fix react-native-gesture-handler RNGestureHandlerModule.kt: (11, 22): Unresolved reference: common 
This is pr [1760](https://hub.fastgit.xyz/software-mansion/react-native-gesture-handler/issues/1760#)

### CODE REVIEW

**Comment by daxiondi** [comment]
> CLA

YES

**Comment by saghul** [comment]
We need package-lock checked in.

More importantly, what is this update fixing? We haven't run into the linked issue AFAICT.

**Comment by daxiondi** [comment]
> 

I got this error when building Android when I was using itsi-meet-stable-jitsi-meet_7287 version, I solved this problem by upgrading the version, so report back to the community

**Comment by daxiondi** [comment]
You can try to rebuild Android, if you don't have time, close pr. Just help the people involved.

File: package.json [hunk 1]
```diff
@@ -99,7 +99,7 @@
     "react-native-default-preference": "1.4.4",
     "react-native-device-info": "8.4.8",
     "react-native-dialog": "9.2.1",
-    "react-native-gesture-handler": "2.1.0",
+    "react-native-gesture-handler": "^2.2.0",
```
Comment (saghul) [inline package.json:102]: Please pin it to the exact version.

**Comment by daxiondi** [comment]
@saghul Do you know how jitsi can limit the length of a meeting in the case of one person? After a long time, clean it out. I searched the forum and couldn't find the relevant answer

**Comment by saghul** [comment]
> @saghul Do you know how jitsi can limit the length of a meeting in the case of one person? After a long time, clean it out. I searched the forum and couldn't find the relevant answer

No, there is no such a feature.

**Comment by daxiondi** [comment]
> > @saghul你知道 jitsi 如何限制一个人的会议时间吗？时间长了，把它清理干净。我搜索了论坛并没有找到相关的答案
> 
> 不，没有这样的功能。
thank you，I use jitsi, and both parties have normal video calls, but jicofo cannot receive the message of leaving after one end of the network is disconnected. Do you have any good ideas?

**Comment by damencho** [comment]
> @saghul Do you know how jitsi can limit the length of a meeting in the case of one person? After a long time, clean it out. I searched the forum and couldn't find the relevant answer

There is a such option, I think, if you use a reservation system. https://jitsi.github.io/handbook/docs/devops-guide/reservation

**Comment by daxiondi** [comment]
> > @saghul Do you know how jitsi can limit the length of a meeting in the case of one person? After a long time, clean it out. I searched the forum and couldn't find the relevant answer
> 
> There is a such option, I think, if you use a reservation system. https://jitsi.github.io/handbook/docs/devops-guide/reservation

Thank you, when one party leaves the meeting because there is no network, jicofo can't detect it. Does jitsi have a similar ping pong? Or some other good idea.

**Comment by damencho** [comment]
> Thank you, when one party leaves the meeting because there is no network, jicofo can't detect it. Does jitsi have a similar ping pong? Or some other good idea.

It is not jicofo, but prosody. And when this is bosh or websockets, the session will timeout in 1 minute. When the session timeouts the participant leaves the room and jicofo sees that.

**Comment by daxiondi** [comment]
> > Thank you, when one party leaves the meeting because there is no network, jicofo can't detect it. Does jitsi have a similar ping pong? Or some other good idea.
> 
> It is not jicofo, but prosody. And when this is bosh or websockets, the session will timeout in 1 minute. When the session timeouts the participant leaves the room and jicofo sees that.

This is not what I see. The bosh I use is in the accelog log of nginx. There is no /http-bind information, but prosody does not capture it, nor does jicofo. The room always exists

**Comment by saghul** [comment]
Please use our community for questions, this PR has nothing to do with any of it.

**Comment by daxiondi** [comment]
> 请使用我们的社区提问，此PR与任何内容无关。

OK, thank you. Do you have any questions about PR?

**Comment by daxiondi** [comment]
> Please use our community for questions, this PR has nothing to do with any of it.

OK, thank you. Do you have any questions about PR?

**Comment by saghul** [comment]
> > Please use our community for questions, this PR has nothing to do with any of it.
> 
> OK, thank you. Do you have any questions about PR?

The CI is failing.

**Comment by daxiondi** [comment]
> > > Please use our community for questions, this PR has nothing to do with any of it.
> > 
> > 
> > OK, thank you. Do you have any questions about PR?
> 
> The CI is failing.

Fix.

**Pull request closed without merging**